runtime/interpreter/value.go

```diff
@@ -2884,10 +2897,16 @@ func (v IntValue) Less(other NumberValue) BoolValue {
 	}
 
 	cmp := v.BigInt.Cmp(o.BigInt)
```

Comment:
Maybe move the bigint comparisons into the getter below. Same for the other operations below

Reply:
Added in: 791fc6caf01c74c52e0491a905cf0bbe2242a300